client/protos/remote_torch.proto

```diff
@@ -0,0 +1,142 @@
+/* Copyright 2022 Mithril Security. All rights reserved.
```

Comment:
Why has the proto file move to the client? Or is this a copy?
Either way, I think it should go to the root, maybe in a protos dir

Reply:
So, I realized that for the CI, it was sometimes failing to read the `../server` path in the `setup.py`.
I think this is because for the publish workflow, it sort of isolates the client directory.
We could but it may still not see it.